packages/react-core/src/utils/InvalidColumnError.js

```diff
@@ -0,0 +1,16 @@
+const NAME = 'InvalidColumnError';
+const MESSAGE_START = `Uncaught ${NAME}:`;
+
+export class InvalidColumnError extends Error {
+  static message =
+    'The column selected for this widget is no longer available in the data source.';
+
+  constructor(message) {
```

Comment:
My proposition would be to change this to

super(`ERR_INVALID_COLUMN: ${message || default}`)

and in is just test for existence of `ERR_INVALID_COLUMN: ` in error message.

This is still _string_ test, but at least based on something that looks like error code and _is_ universal and doesn't depend on particular details like adding "Uncaught..." by particular runtime.